tests/contrib/django/test_connection.py

```diff
@@ -36,3 +40,14 @@ def test_connection(self):
         eq_(span.get_tag('django.db.alias'), 'default')
         eq_(span.get_tag('sql.query'), 'SELECT COUNT(*) AS "__count" FROM "auth_user"')
         assert start < span.start < span.start + span.duration < end
+
+    @override_settings(DATADOG_TRACE=NEW_SETTINGS)
```

Comment:
Yes, unfortunately the cursor is already initialized with the previous setting (that's why I remembered that it wasn't so trivial!). Let me take a look and see if we can do something here.

Thank you!